packages/seacas/libraries/ioss/src/Ioss_ElementTopology.h

```diff
@@ -117,6 +129,8 @@ namespace Ioss {
     bool operator!=(const Ioss::ElementTopology &rhs) const;
     bool equal(const Ioss::ElementTopology &rhs) const;
 
+    virtual ElementPermutation *permutation() const = 0;
```

Comment:
As mentioned in another comment, I think we might be able to make this be a non-virtual function at this level instead of overriding in each derived class.

This would then add the permutation capability with no modifications to the derived element classes...  The super element might be a little more difficult...

Reply:
Let me try another revision based on your review and I'll keep you updated